docs/docs/cmd/graph/changelog/changelog-list.md

```diff
@@ -51,3 +51,45 @@ Get all changes within Microsoft Graph that happend between _2021-01-01_ and _20
 ```sh
 m365 graph changelog list --startDate '2021-01-01' --endDate '2021-05-01'
 ```
+
+## Response
+
+
+=== "JSON"
+    ```json
+    [
+      {
+        "guid": "f5545eaf-7e2f-424a-b4cd-61d5a95cc44fbeta",
+        "category": "beta",
+        "title": "Personal contacts",
+        "description": "Added mobilePhone property to personal contacts entity-set.\\\n",
+        "pubDate": "2015-12-01T00:00:00.000Z"
+      },
+      {
+        "guid": "a0eccf7b-3efb-4c5f-bb1c-4049202b1e0fbeta",
+        "category": "beta",
+        "title": "Calendar",
+        "description": "Added eventMessageRequest subtype of eventMessage and startDateTime, endDateTime, location, type, recurrence and isOutOfDate properties to eventMessage type.\\\n",
+        "pubDate": "2015-12-01T00:00:00.000Z"
+      }
+    ]
+    ```
+
+=== "Text"
+
+    ```text
+    category title description
+    ---
```

Comment:
Seems like something went wrong with the heading of this output. Normally all headers are underlined.

![image](https://user-images.githubusercontent.com/11723921/205406633-b602b754-9520-4cd4-b304-ba5f2a8d3b20.png)

Also: let's show only 1 object (the same as the JSON output).